README.md

```diff
@@ -20,463 +21,441 @@ This project now supports Swift 2, if you still require Swift 1.2 you can use th
+
 ```swift
- //GET DEVICE
-        ExpSwift.getDevice("8930ff64-1063-4a03-b1bc-33e1ba463d7a").then { (device: Device) -> Void  in
-                println(device.get("name"))
-            }.error { error in
-                println(error)
-        }
+ExpSwift.getDevice("8930ff64-1063-4a03-b1bc-33e1ba463d7a").then { (device: Device) -> Void  in
+         debugPrint(device.get("name"))
+        }.error { error in
+         debugPrint(error)
+}
 ```
 
-### ExpSwift.findDevices(params:[String:AnyObject])
+**`ExpSwift.findDevices(params:[String:AnyObject])`**
+
 Query for multiple devices. Resolves to an array of [Device Objects](#device-object).
+
 ```swift
- //GET DEVICES
-        ExpSwift.findDevices(["limit":10, "skip":0, "sort":"name"]).then { (devices: SearchResults<Device>) -> Void  in
+ExpSwift.findDevices(["limit":10, "skip":0, "sort":"name"]).then { (devices: SearchResults<Device>) -> Void  in
             for device in devices.getResults() {
-                println(device.get("name"))
+                debugPrint(device.get("name"))
             }
         }.error { error in
-            println(error)
-        }
-```
-### ExpSwift.getCurrentExperience()
-Get the current experience. Resolves to an [Experience Object](#experience-object).
-```swift
-//GET CURRENT EXPERIENCE
-        ExpSwift.getCurrentExperience().then { experience -> Void  in
-            println(experience.get("name"))
-            }.error { error in
-                println(error)
-        }
-```
-### ExpSwift.getExperience(uuid:String)
-Get a single experience by UUID. Resolves to a [Experience Object](#experience-object).
-```swift
-//GET EXPERIENCE
-        ExpSwift.getExperience("58dc59e4-a44c-4b6e-902b-e6744c09d933").then { (experience: Experience) -> Void  in
-            println(experience.get("name"))
-        }.error { error in
-                println(error)
-        }
+            debugPrint(error)
+}
 ```
-### ExpSwift.findExperiences(params:[String:AnyObject])
-Query for multiple experiences. Resolves to an array of [Experience Objects](#experience-object).
-```swift
- //GET EXPERIENCES
-        ExpSwift.findExperiences(["limit":10, "skip":0, "sort":"name"]).then { (experiences: SearchResults<Experience>) -> Void  in
-            for experience in experiences.getResults() {
-                println(experience.get("name"))
-            }
-        }.error { error in
-                println(error)
-        }
 
-```
+## Things
 
-### ExpSwift.getLocation(uuid:String)
-Get a single location by UUID. Resolves to a [Location Object](#location-object).
-```swift
- //GET LOCATION
-        ExpSwift.getLocation("3e2e25df-8324-4912-91c3-810751f527a4").then { (location: Location) -> Void  in
-            println(location.get("name"))
-            }.error { error in
-                println(error)
-        }
+**`ExpSwift.getThing(uuid:String)`**
 
-```
+Get a single thing by UUID. Resolves to a [Thing Object](#thing-object).
```

Comment:
These links don't seem to work. There is no thing-object or device-object, etc to go to in this doc.